### PR TITLE
html lint

### PR DESCRIPTION
**Pull request by iondrimba**
Added Html lint rules via https://www.npmjs.com/package/htmllint
To be used with front-end stack

### CODE REVIEW

File: .htmllintrc [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+  "attr-bans": [
+    "align",
+    "background",
+    "bgcolor",
+    "border",
+    "dynsrc",
+    "id",
```
Comment (eduardojmatos) [inline .htmllintrc]: I think that `id` can be "authorized" to preserve same behaviours of our old code base

File: .htmllintrc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "longdesc",
+    "lowsrc",
+    "onclick",
+    "ondblclick",
+    "onload",
+    "marginwidth",
+    "marginheight",
+    "scrolling",
+    "style",
+    "width"
```
Comment (eduardojmatos) [inline .htmllintrc]: @iondrimba it's a good practice to use width on `img` tag :/

File: .htmllintrc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  "attr-name-style": "dash",
+  "attr-no-dup": true,
+  "attr-no-unsafe-chars": true,
+  "attr-quote-style": "double",
+  "attr-req-value": true,
+  "class-no-dup": true,
+  "class-style": "dash",
+  "csslint": false,
+  "doctype-first": false,
+  "doctype-html5": false,
```
Comment (eduardojmatos) [inline .htmllintrc]: @iondrimba this role can be true =D

File: .htmllintrc [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  "fig-req-figcaption": true,
+  "focusable-tabindex-style": false,
+  "head-req-title": true,
+  "href-style": false,
+  "html-req-lang": true,
+  "id-class-ignore-regex": false,
+  "id-class-no-ad": true,
+  "id-class-style": "underscore",
```
Comment (eduardojmatos) [inline .htmllintrc]: @iondrimba we can mantain dash pattern here too

File: .htmllintrc [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  "line-max-len-ignore-regex": "/href/g",
+  "page-title": true,
+  "spec-char-escape": true,
+  "table-req-caption": false,
+  "table-req-header": true,
+  "tag-bans": [
+    "b",
+    "i",
+    "keygen",
+    "style"
```
Comment (eduardojmatos) [inline .htmllintrc]: @iondrimba AMP pages will cry hehe

File: .htmllintrc [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  ],
+  "tag-close": true,
+  "tag-name-lowercase": true,
+  "tag-name-match": true,
+  "tag-self-close": "never",
+  "text-escape-spec-char": true,
+  "title-max-len": 60,
```
Comment (eduardojmatos) [inline .htmllintrc]: @iondrimba same of our pages have more than 60 letters (sample: https://www.getninjas.com.br/reformas-e-reparos/pedreiro/sp/sao-paulo/orcamentos/4485663-preciso-reformar-uma-casa-antiga-e).

120 could be a nice number

File: .htmllintrc (hunk 5 above)
Comment (williamcosta) [reply .htmllintrc]: @iondrimba `style` tag bans? Critical Path cry.

**Pull request merged**